modules/nw-ingress-sharding.adoc

```diff
@@ -0,0 +1,20 @@
+// Module included in the following assemblies:
+//
+// * configuring_ingress_cluster_traffic/configuring-ingress-cluster-traffic-ingress-controller.adoc
+// * ingress-operator.adoc
+
+[id="nw-ingress-sharding_{context}"]
+= Ingress Controller sharding
+
+As the primary mechanism for traffic to enter the cluster the demands on the Ingress Controller, or router, can be significant. As cluster administrator, you can shard the routes to:
```

Comment:
"to enter the cluster the demands" - something is off here, I think it needs a comma:

> As the primary mechanism for traffic to enter the cluster, the demands on the Ingress Controller